src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/pdf/PdfBoxServiceException.java

```diff
@@ -0,0 +1,41 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service.pdf;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.Serial;
+
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+import static uk.gov.hmcts.ethos.replacement.docmosis.service.pdf.et3.ET3FormConstants.STRING_EMPTY;
+
+/**
+ * Is thrown when an exception occurs whiles converting case data in pdf in {@link PdfBoxService}.
+ * for example an IOException is encountered while loading pdf template
+ */
+@Slf4j
+public class PdfBoxServiceException extends Exception {
```

Comment:
Rather than having this called as a PdfBoxServiceException, is it worth renaming it as you've used it for other things which aren't directly tied to PdfBox